src/Common/ScrollToTop/ScrollToTop.jsx

```diff
@@ -4,7 +4,7 @@ import { withRouter } from "react-router-dom";
 class ScrollToTop extends Component {
   componentDidUpdate(prevProps) {
     if (this.props.location !== prevProps.location) {
-      window.scrollTo(0, 0);
```

Comment:
this was the fix for auto-scrolling to the top of the page when navigating between pages

Reply:
Shall we put a comment to explain this?